### PR TITLE
Refresh the foundation page

### PR DESCRIPTION
**Pull request by grahamc**
Expand the role of the foundation, scale of the infrastructure.

Looks like this:

image

### CODE REVIEW

File: nixos/foundation.tt [hunk 1]
```diff
@@ -1,29 +1,47 @@
...
 <p>The NixOS Foundation is a registered non-profit organisation at the Chamber of
 commerce (Kamer van Koophandel) in Utrecht, The Netherlands. The KvK number is 63520583.
 </p>
 
-<p>The purpose of the foundation, is to help support the infrastructure and projects
-that are implementing the purely functional deployment model, such as NixOS, Nix
-and NixOps.</p>
-
-<p>There are currently 3 board members:</p>
+<p>There board member consists of:</p>
```
Comment (samueldr) [inline nixos/foundation.tt]: > There board member

I don't have strong opinions about a replacement, but I do know this isn't cromulent english.

Comment (grahamc) [reply nixos/foundation.tt]: Thanks, fixed

File: nixos/foundation.tt [hunk 2]
```diff
@@ -1,29 +1,47 @@
...
-<h2>General</h2>
+<div class="row-fluid">
+<div class="span6">
+<h2>About The Foundation</h2>
+
+<p>The mission of the foundation is to support the Nix ecosystem's
+infrastructure, and projects implementing the purely functional
+deployment model.</p>
+
+<p>In particular, the foundation supports Nix, Nixpkgs, NixOS, and
```
Comment (edolstra) [inline nixos/foundation.tt]: -> `supports Nix, Nixpkgs and NixOS, and operates`.

Comment (grahamc) [reply nixos/foundation.tt]: Fixed

**Comment by grahamc** [comment]
Thanks!

**Pull request merged**